actionpack/lib/action_dispatch/routing.rb

```diff
@@ -243,9 +243,9 @@ module ActionDispatch
   #
   #   rails routes
   #
-  # Target specific controllers by prefixing the command with <tt>-c</tt> option. Use
-  # <tt>--expanded</tt> to turn on the expanded table formatting mode.
-  #
+  # Target specific controller by prefixing the command with <tt>-c</tt> option,
```

Comment:
Let's strike the 2 `option` occurences. The sentences sound better without them.

It should be: `Target a specific`.

Reply:
> by prefixing the command with <tt>-c</tt> option

That doesn't sound right; options are passed after the command. I'd ✂️ "by prefixing the command" here.